public/docs/ts/latest/guide/production-deploy.jade

```diff
@@ -0,0 +1,480 @@
+  a site that hosts _npm packages_,
+  and loads them from the web directly.
+  There are other service providers that do the same thing.
+
+  If you are unwilling or unable to load packages from the open web, 
+  the inventory in `systemjs.config.server.js` identifies the files and folders that 
+  you would copy to a library folder on the server. 
+  Then change the config's  `'npm'` path to point to that folder.
+
+  ### Practice with an example
+
+  The following trivial router sample app shows these changes.
+
++makeTabs(
+  `production-deploy/ts/index.html,
+   production-deploy/ts/systemjs.config.server.js,
+   production-deploy/ts/app/main.ts,
+   production-deploy/ts/app/app.module.ts,
+   production-deploy/ts/app/app.component.ts,
+   production-deploy/ts/app/crisis-list.component.ts,
+   production-deploy/ts/app/hero-list.component.ts
+  `, 
+  null,
+  `index.html,
+   systemjs.config.server.js,
+   app/main.ts,
+   app/app.module.ts,
+   app/app.component.ts,
+   app/crisis-list.component.ts,
+   app/hero-list.component.ts
+  `)
+
+:marked
+  Practice with this sample before attempting these techniques on your application.
+
+  1. Follow the [setup instructions](../guide/setup.html "Angular QuickStart setup") for creating a new project
+  named <ngio-ex path="simple-deployment"></ngio-ex>.
+
+  1. Add the "Simple deployment" sample files shown above.
+
+  1. Run it with `npm start` as you would any project.
+
+  1. Inspect the network traffic in the browser developer tools. 
+  Notice that it loads all packages from the web.
+  You could delete the `node_modules` folder and the app would still run 
+  (although you wouldn't be able to recompile or launch `lite-server`
+  until you restored it).
+
+  1. Deploy the sample to the server (minus the `node_modules` folder!).
+
+  When you have that working, try the same process on your application.
+
+a#optimize
+.l-main-section
+:marked
+  ## Optimize for production
+
+  Although deploying directly from the development environment works, it's far from optimal.
+  
+  The client makes many small requests for individual application code and template files,
+  a fact you can quickly confirm by looking at the network tab in a browser's developer tools.
+  Each small file download can spend more time communicating with the server than tranfering data.
+
+  Development files are full of comments and whitespace for easy reading and debugging.
+  The browser downloads entire libraries, instead of just the parts the app needs.
+  The volume of code passed from server to client (the "payload") 
+  can be significantly larger than is strictly necessary to execute the application.
+
+  The many requests and large payloads mean
+  the app takes longer to launch than it would if you optimized it.
+  Several seconds may pass (or worse) before the user can see or do anything userful.
+  
+  Does it matter? That depends upon business and technical factors you must evaluate for yourself.
```

Comment:
On the docs meetings a couple of real world scenarios came up. Deploying demos was one, and for some projects there is no time/feature budget to actually produce an optimized deployment since the business does not need it.